BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/rest/controllers/PendingSubmissionResource.java

```diff
@@ -0,0 +1,57 @@
+package uk.ac.ebi.biostd.webapp.application.rest.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+import uk.ac.ebi.biostd.authz.User;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionDto;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionListFiltersDto;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionListDto;
+import uk.ac.ebi.biostd.webapp.application.rest.service.PendingSubmissionService;
+
+@AllArgsConstructor
+@RestController
+@PreAuthorize("isAuthenticated()")
+public class PendingSubmissionResource {
+
+    private final PendingSubmissionService pendingSubmissionService;
+
+    @GetMapping("/submissions/pending")
+    public @ResponseBody
+    PendingSubmissionListDto getSubmissions(PendingSubmissionListFiltersDto filters,
+                                            @AuthenticationPrincipal User user) {
+        return pendingSubmissionService.getSubmissionList(filters, user);
+    }
```

Comment:
correct formatting (according to google standards use in this project will be )

```
    @GetMapping("/submissions/pending")
    @ResponseBody
    public PendingSubmissionListDto getSubmissions(PendingSubmissionListFiltersDto filters,
            @AuthenticationPrincipal User user) {
        return pendingSubmissionService.getSubmissionList(filters, user);
    }
```

Reply:
Done